tools/mmp/driver.cs

```diff
@@ -555,7 +560,7 @@ static void Pack (IList<string> unprocessed)
 				GatherAssemblies ();
 				CheckReferences ();
 
-				if (!resolved_assemblies.Exists (f => Path.GetExtension (f).ToLower () == ".exe"))
+				if (!resolved_assemblies.Exists (f => Path.GetExtension (f).ToLower () == ".exe") && !is_extension)
 					throw new MonoMacException (79, true, "No executable was copied into the app bundle.  Please contact 'support@xamarin.com'", "");
```

Comment:
Would change the if to have !is_extension first to short cirtuit the && and not do the line expression.